app/helpers/wpcc/your_results_page_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  module YourResultsPageHelper
+    TAX_RELIEF_THRESHOLD_RATE = 11_500
+
+    def earning_below_tax_relief_threshold?(salary)
+      salary.to_i < TAX_RELIEF_THRESHOLD_RATE
+    end
+  end
+end
```

Comment:
If we are sticking to using the presenter to present information I'm fine with it. I have no strong or fussy inclination to the use of either presenters, decorators or helpers.